comparators/added_place.js

```diff
@@ -3,8 +3,9 @@
 module.exports = addedPlace;
 
 function addedPlace(newVersion, oldVersion) {
+  var result = false;
```

Comment:
Thanks @lukasmartinelli , that sounds like a great idea 💯 and I should totally do that.

>I think for primitive values as false it is not a code duplication to return false

Yeah I avoided that thinking of it as code duplication. But yes, it does make difficult reading result value. Well said 👍